modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/service/impl/FriendlyURLEntryLocalServiceImpl.java

```diff
@@ -497,7 +569,8 @@ public void setMainFriendlyURLEntry(FriendlyURLEntry friendlyURLEntry) {
 	@Override
 	public FriendlyURLEntry updateFriendlyURLEntry(
 			long friendlyURLEntryId, long classNameId, long classPK,
-			String defaultLanguageId, Map<String, String> urlTitleMap)
+			String defaultLanguageId, Map<String, String> urlTitleMap,
+			ServiceContext serviceContext)
```

Comment:
@adolfopa, we should mark this as a breaking change right?

Reply:
We should keep the old method around, and delegate on the new one. As the service interface is a provider type, that won't be a breaking change.